This is not final color of the buttons. :D

LukaszSokolowski opened the pull request:
+ oval corners
+ visible frames

LukaszSokolowski committed:
This is not final color of the buttons. :D
+ oval corners
+ visible frames

yakoobs commented on Elements/Elements/ViewControllers/GameViewController.swift:
```diff
@@ -39,7 +39,11 @@ extension GameViewController {
+            button.layer.cornerRadius = 5
+            button.layer.borderWidth = 1
+            button.layer.borderColor = UIColor.black.cgColor
+            
```
It's fine but take a look at [https://github.com/rhodgkins/UIView-IBInspectable/blob/master/IBInspectables.swift)](https://github.com/rhodgkins/UIView-IBInspectable/blob/master/IBInspectables.swift)
Read about `IBInspectable` - you can set values like the `cornerRadius`, `borderWidth` and `borderColor` from **Interface Builder**! Read also about extension for Swift classes.
Finally create an extension for `UIView` (`UIButton` inherits from `UIView`) like in the example above and set those values for every single button in the Interface Builder.
Your solution is absolutely fine. But do this exercise just for practise purposes.

yakoobs replied:
Here u can find an article [http://nshipster.com/ibinspectable-ibdesignable/](http://nshipster.com/ibinspectable-ibdesignable/) (Swift 2!)

LukaszSokolowski committed:
Added menu and IBInspectable extension

yakoobs reviewed (commented):
Looks good to me!

LukaszSokolowski committed:
- Points label
- Attempts label
- Game Over View Controller

I’m not sure that the code where we count the points and attempts should be in ViewController instead of Model

yakoobs commented on Elements/Elements/GameOverViewController.swift:
```diff
@@ -0,0 +1,35 @@
+        // Get the new view controller using segue.destinationViewController.
+        // Pass the selected object to the new view controller.
+    }
+    */
```
Do not commit commented code like this method.

yakoobs commented on Elements/Elements/MenuViewController.swift:
```diff
@@ -0,0 +1,41 @@
+    
+
+    
+
```
Remove please empty methods end empty lines
